src/components/ServiceList.jsx

```diff
@@ -1,22 +1,67 @@
 import { Alert } from "@baltimorecounty/dotgov-components";
 import FilterList from "../components/FilterList";
-import { MostPopularServiceIconStyles } from "../styles";
-import React from "react";
-import ServiceIconLink from "../components/ServiceIconLink";
+import React, { useState } from "react";
+import ServiceIconLink from "./ServiceIconLink";
+import SearchCollapse from "./SearchCollapse";
 import useServices from "../hooks/useServices";
+import {
+  MostPopularServiceIconStyles,
+  LegendText,
+  CollapseContainer
+} from "../styles";
+import Search from "./Search";
 
 const ServiceList = () => {
   const { hasError, serviceItems = [], isLoading } = useServices();
+  const [isChecked, setChecked] = useState(false);
+  const [searchedItems, setSearchedItems] = useState([]);
+  const [searchText, setSearchText] = useState([]);
+  const [isFiltering, setIsFiltering] = useState(false);
 
-  let legendText = {
-    fontStyle: "italic",
-    marginLeft: "5px",
-    fontWeight: "900"
+  const filterItems = (services, searchText) => {
+    return services.filter(item => {
+      const { name, department } = item;
+      return (
+        name.toLowerCase().indexOf(searchText.toLowerCase()) !== -1 ||
+        department.toLowerCase().indexOf(searchText.toLowerCase()) !== -1
+      );
+    });
   };
 
-  let flex = {
-    display: "flex",
-    flexDirection: "row"
+  const filterByPopularity = item => {
```

Comment:
when you have a tenary that results in a true and false you can simplify to just condition

```js
const filterByPopularity = item => item.rank > 0;
```

Reply:
done